docs/en/v0.3/user-guide/monitoring.md

```diff
@@ -0,0 +1,136 @@
+# Metrics
+
+To collect GreptimeDB metrics using Prometheus
+
+## Binary
+
+Download the GreptimeDB binary, you can use the following command:
+```
+curl -fsSL \
+  https://raw.githubusercontent.com/greptimeteam/greptimedb/develop/scripts/install.sh | sh
+```
+
+Run GreptimeDB:
+```
+./greptime standalone start
+```
+
+[The official documentation for prometheus](https://prometheus.io/download/) download the binary:
+```
+tar xvfz prometheus-*.tar.gz
+cd prometheus-*
+```
+
+Write a Prometheus configuration file and save it as prometheus.yml:
+```
+global:
+  scrape_interval: 10s 
+
+scrape_configs:
+  - job_name: 'greptimedb'
+    static_configs:
+      - targets: ['localhost:4000']
+```
+
+Run prometheus:
+```
+./prometheus --config.file=prometheus.yml
+```
+
+Access Prometheus by entering `localhost:9090` in your web browser.
+
+
+## Docker
+
+Run GreptimeDB:
+```
+docker run -p 4000-4003:4000-4003 \
+  -p 4242:4242 -v "$(pwd)/greptimedb:/tmp/greptimedb" \
+  --name greptime --rm \
+  greptime/greptimedb standalone start \
+  --http-addr 0.0.0.0:4000 \
+  --rpc-addr 0.0.0.0:4001 \
+  --mysql-addr 0.0.0.0:4002 \
+  --postgres-addr 0.0.0.0:4003 \
+  --opentsdb-addr 0.0.0.0:4242
+```
```

Comment:
remove the content above and add a link to getting-started/try-out-greptimeDB/docker